### ISSUE TITLE
App Crash On Theme Editor 

### ISSUE THREAD

**Issue by GoRaN909** [florisboard/florisboard#1838]
#### Short description
I tried to create a custom theme using existing default floris theme, When i want to change the common shape great than 33dp the app is crash suddenly you can't use more than 33dp for corners of buttons shape. 

#### Steps to reproduce
1. Go to 'Theme'
2. Click on 'Create new theme'
3. Scroll down to 'Common shape'
4. See crash

#### Environment information
- FlorisBoard Version: 0.3.16-beta01 (81)
- Device: Samsung SM-G781B
- Android: 12 (cn=S sdk=31)

#### Features enabled
```kt
smartbar = true
suggestions = false
suggestions_clipboard = false
suggestions_next_word = false
glide = false
clipboard_internal = true
clipboard_history = true
```

#### Attached stacktrace files
<details>
<summary>1652137212120.stacktrace</summary>

```
java.lang.IllegalArgumentException: The percent should be in the range of [0, 100]
	at androidx.compose.foundation.shape.PercentCornerSize.<init>(CornerSize.kt:95)
	at androidx.compose.foundation.shape.CornerSizeKt.CornerSize(CornerSize.kt:82)
	at androidx.compose.foundation.shape.RoundedCornerShapeKt.RoundedCornerShape(RoundedCornerShape.kt:195)
	at dev.patrickgold.florisboard.app.settings.theme.SnyggValueIconKt.alwaysPercentShape(SnyggValueIcon.kt:166)
	at dev.patrickgold.florisboard.app.settings.theme.SnyggValueIconKt.SnyggValueIcon(SnyggValueIcon.kt:104)
	at dev.patrickgold.florisboard.app.settings.theme.SnyggValueIconKt.SnyggValueIcon(SnyggValueIcon.kt:134)
	at dev.patrickgold.florisboard.app.settings.theme.ThemeEditorScreenKt$ThemeEditorScreen$1$5$4$2$1$1$4.invoke(ThemeEditorScreen.kt:311)
	at dev.patrickgold.florisboard.app.settings.theme.ThemeEditorScreenKt$ThemeEditorScreen$1$5$4$2$1$1$4.invoke(ThemeEditorScreen.kt:311)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:107)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:34)
	at androidx.compose.runtime.RecomposeScopeImpl.compose(RecomposeScopeImpl.kt:140)
	at androidx.compose.runtime.ComposerImpl.recomposeToGroupEnd(Composer.kt:2158)
	at androidx.compose.runtime.ComposerImpl.skipToGroupEnd(Composer.kt:2427)
	at androidx.compose.material.TextKt.ProvideTextStyle(Text.kt:266)
	at androidx.compose.material.ListItemKt$applyTextStyle$1$1.invoke(ListItem.kt:425)
	at androidx.compose.material.ListItemKt$applyTextStyle$1$1.invoke(ListItem.kt:424)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:107)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:34)
	at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:228)
	at androidx.compose.material.ListItemKt$applyTextStyle$1.invoke(ListItem.kt:424)
	at androidx.compose.material.ListItemKt$applyTextStyle$1.invoke(ListItem.kt:423)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:107)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:34)
	at androidx.compose.material.TwoLine$ListItem$1$4.invoke(ListItem.kt:267)
	at androidx.compose.material.TwoLine$ListItem$1$4.invoke(ListItem.kt:261)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:107)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:34)
	at androidx.compose.runtime.RecomposeScopeImpl.compose(RecomposeScopeImpl.kt:140)
	at androidx.compose.runtime.ComposerImpl.recomposeToGroupEnd(Composer.kt:2158)
	at androidx.compose.runtime.ComposerImpl.skipToGroupEnd(Composer.kt:2427)
	at androidx.compose.material.ListItemKt.OffsetToBaselineOrCenter-Kz89ssw(ListItem.kt:415)
	at androidx.compose.material.ListItemKt.access$OffsetToBaselineOrCenter-Kz89ssw(ListItem.kt:1)
	at androidx.compose.material.TwoLine.ListItem(ListItem.kt:255)
	at androidx.compose.material.ListItemKt.ListItem(ListItem.kt:96)
	at dev.patrickgold.jetpref.material.ui.JetPrefListItemKt.JetPrefListItem(JetPrefListItem.kt:58)
	at dev.patrickgold.florisboard.app.settings.theme.ThemeEditorScreenKt$ThemeEditorScreen$1$5$4$2$1.invoke(ThemeEditorScreen.kt:302)
	at dev.patrickgold.florisboard.app.settings.theme.ThemeEditorScreenKt$ThemeEditorScreen$1$5$4$2$1.invoke(ThemeEditorScreen.kt:276)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:116)
	at androidx.compose.runtime.internal.ComposableLambdaImpl$invoke$1.invoke(ComposableLambda.jvm.kt:127)
	at androidx.compose.runtime.internal.ComposableLambdaImpl$invoke$1.invoke(ComposableLambda.jvm.kt:127)
	at androidx.compose.runtime.RecomposeScopeImpl.compose(RecomposeScopeImpl.kt:140)
	at androidx.compose.runtime.ComposerImpl.recomposeToGroupEnd(Composer.kt:2158)
	at androidx.compose.runtime.ComposerImpl.skipToGroupEnd(Composer.kt:2427)
	at dev.patrickgold.florisboard.lib.compose.FlorisCardsKt.FlorisOutlinedBox-wK_Y8yA(FlorisCards.kt:297)
	at dev.patrickgold.florisboard.app.settings.theme.ThemeEditorScreenKt$ThemeEditorScreen$1$5$4$invoke$$inlined$items$default$2.invoke(LazyDsl.kt:275)
	at dev.patrickgold.florisboard.app.settings.theme.ThemeEditorScreenKt$ThemeEditorScreen$1$5$4$invoke$$inlined$items$default$2.invoke(LazyDsl.kt:100)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:135)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:34)
	at androidx.compose.foundation.lazy.list.LazyListScopeImpl$items$1$1.invoke(LazyListScopeImpl.kt:41)
	at androidx.compose.foundation.lazy.list.LazyListScopeImpl$items$1$1.invoke(LazyListScopeImpl.kt:41)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:107)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:34)
	at androidx.compose.runtime.CompositionLocalKt.CompositionLocalProvider(CompositionLocal.kt:228)
	at androidx.compose.runtime.saveable.SaveableStateHolderImpl.SaveableStateProvider(SaveableStateHolder.kt:84)
	at androidx.compose.foundation.lazy.layout.LazyLayoutItemContentFactory$CachedItemContent$content$1.invoke(LazyLayoutItemContentFactory.kt:103)
	at androidx.compose.foundation.lazy.layout.LazyLayoutItemContentFactory$CachedItemContent$content$1.invoke(LazyLayoutItemContentFactory.kt:94)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:107)
	at androidx.compose.runtime.internal.ComposableLambdaImpl.invoke(ComposableLambda.jvm.kt:34)
	at androidx.compose.runtime.RecomposeScopeImpl.compose(RecomposeScopeImpl.kt:140)
	at androidx.compose.runtime.ComposerImpl.recomposeToGroupEnd(Composer.kt:2158)
	at androidx.compose.runtime.ComposerImpl.skipCurrentGroup(Composer.kt:2404)
	at androidx.compose.runtime.ComposerImpl$doCompose$2$5.invoke(Composer.kt:2585)
	at androidx.compose.runtime.ComposerImpl$doCompose$2$5.invoke(Composer.kt:2571)
	at androidx.compose.runtime.SnapshotStateKt__DerivedStateKt.observeDerivedStateRecalculations(DerivedState.kt:247)
	at androidx.compose.runtime.SnapshotStateKt.observeDerivedStateRecalculations(Unknown Source:1)
	at androidx.compose.runtime.ComposerImpl.doCompose(Composer.kt:2571)
	at androidx.compose.runtime.ComposerImpl.recompose$runtime_release(Composer.kt:2547)
	at androidx.compose.runtime.CompositionImpl.recompose(Composition.kt:620)
	at androidx.compose.runtime.Recomposer.performRecompose(Recomposer.kt:786)
	at androidx.compose.runtime.Recomposer.access$performRecompose(Recomposer.kt:105)
	at androidx.compose.runtime.Recomposer$runRecomposeAndApplyChanges$2$2.invoke(Recomposer.kt:456)
	at androidx.compose.runtime.Recomposer$runRecomposeAndApplyChanges$2$2.invoke(Recomposer.kt:425)
	at androidx.compose.ui.platform.AndroidUiFrameClock$withFrameNanos$2$callback$1.doFrame(AndroidUiFrameClock.android.kt:34)
	at androidx.compose.ui.platform.AndroidUiDispatcher.performFrameDispatch(AndroidUiDispatcher.android.kt:109)
	at androidx.compose.ui.platform.AndroidUiDispatcher.access$performFrameDispatch(AndroidUiDispatcher.android.kt:41)
	at androidx.compose.ui.platform.AndroidUiDispatcher$dispatchCallback$1.doFrame(AndroidUiDispatcher.android.kt:69)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1106)
	at android.view.Choreographer.doCallbacks(Choreographer.java:866)
	at android.view.Choreographer.doFrame(Choreographer.java:792)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1092)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:313)
	at android.app.ActivityThread.main(ActivityThread.java:8663)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:567)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)
	Suppressed: kotlinx.coroutines.DiagnosticCoroutineContextException: [androidx.compose.runtime.PausableMonotonicFrameClock@3c11333, StandaloneCoroutine{Cancelling}@980e6f0, AndroidUiDispatcher@fdfa469]

```
</details>

**Comment by patrickgold** [comment]
Can reproduce this issue, thus will re-open it and fix it now.

**Comment by patrickgold** [comment]
Above linked PR is a fix for this issue, the preview does not cause a crash anymore for values >= 34dp. Will be released in 0.3.16-beta04 soon, thanks for reporting!

**Comment by GoRaN909** [comment]
_Thank you @patrickgold for fix this bug I tested the debug apk the error was gone :)_
